### ISSUE TITLE
Enable/disable CC

### ISSUE THREAD

**Issue by bfelten1948** [yuliskov/SmartTube#1924]
When I click on disable CC I really don't want to first select language/style/size, and *then* go to the next level where I select language, I just want it turned off (like in the original YT).

Similarly, when clicking on enable, and there's only one to choose from, yeah, that's the one I want (like in the original YT).

Other than that, this is a genuinely awesome app, I don't know how I ever put up with the original. Kudos guys!

**Comment by Jamie-Landeg-Jones** [comment]
Yes! I just came here to request the same thing!

I often find myself needing to enable or disable subtitles depending on the video, and a quick toggle button would be most useful, just like the one that already exists for sponsorblock - and I doubt anyone has to toggle that on/off all that often!

**Comment by garret** [comment]
I agree it would be nice. Started to use the app recently and surprised no one has this before :)

**Comment by bfelten1948** [comment]
Let's not forget that just a few months ago, the app was updated with the extra level (the one where we first have to select style/size and so of the CC that we just want to disable) before getting to the level, where we eventually could *select* the language as no language at all. Who came up with this stupid improvement?

**Comment by jack5432109** [comment]
@yuliskov this issue is brought up at https://github.com/yuliskov/SmartTubeNext/issues/1908 and https://github.com/yuliskov/SmartTubeNext/issues/1768
